index.bs

```diff
@@ -478,21 +488,26 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
      * entry's [=attribution source/attribution destination=] and |attributionDestination| are equal.
      * entry's [=attribution source/reporting endpoint=] and |trigger|'s [=attribution source/reporting endpoint=] are equal.
      * entry's [=attribution source/expiry=] is greater than the current time.
+1. If |matchingSources| is empty, return.
 1. Set |matchingSources| to the result of [=list/sort in descending order|sorting=] |matchingSources|
     in descending order, with |a| being less than |b| if any of the following are true:
       * |a|'s [=attribution source/priority=] is less than |b|'s [=attribution source/priority=].
       * |a|'s [=attribution source/priority=] is equal to |b|'s [=attribution source/priority=] and |a|'s 
          [=attribution source/source time=] is less than |b|'s [=attribution source/source time=].
 1. Let |sourceToAttribute| be the first item in |matchingSources|.
+1. If |trigger|'s [=attribution trigger/dedup key=] is not null and |sourceToAttribute|'s
+    [=attribution source/dedup keys=] [=list/contains=] it, return.
```

Comment:
I think the rest of the spec refers to struct's fields by their values rather than the field.

```suggestion
    [=attribution source/dedup keys=] [=list/contains=] the value, return.
```